lib/models/device.coffee

```diff
@@ -2099,6 +2099,26 @@ getDeviceModel = (deps, opts) ->
 
 		return timeSince(lce)
 
+	###*
+	# @summary Get the composite version number of the OS running on a device
```

Comment:
Maybe we need to be more concrete of what "composite" means?

Something like `Get the OS version number and variant running on a device` (please redefine it to whatever you think is clearer).